src/flowcean/environments/rosbag.py

```diff
@@ -0,0 +1,84 @@
+from os import listdir
+from pathlib import Path
+from typing import Self, override
+
+import polars as pl
+from rosbags.dataframe import get_dataframe
+from rosbags.highlevel import AnyReader
+from rosbags.typesys import Stores, get_types_from_msg, get_typestore
+
+from flowcean.core.environment import OfflineEnvironment
+
+
+def guess_msgtype(path: Path) -> str:
```

Comment:
Make a static function of the `RosbagEnvironment`.

Reply:
done